frontend/src/i18n/en-US.json

```diff
@@ -202,6 +201,18 @@
   "licenseAlert.licenseExpired.user.description": "Your Cryptomator Hub license has expired. Please inform a Hub administrator to renew the license.",
   "licenseAlert.button": "Admin Section",
 
+  "legacyDeviceList.title": "Legacy Devices",
+  "legacyDeviceList.description": "These devices were created with an older version of Hub and will not be able to unlock vaults in a future update. If you no longer need them, please remove them. If you are still using them, please update Cryptomator Desktop and your devices will be migrated to actively supported devices.",
```

Comment:
```suggestion
  "legacyDeviceList.description": "These devices were created with an older version of Hub and need to be updated to unlock vaults in future versions. If you no longer need them, please remove them. Otherwise, please update Cryptomator on that device. Your next unlock will trigger the update automatically.",
```

Reply:
Fixed in 00bdeb1